src/ci/docker/host-x86_64/mingw-check/Dockerfile

```diff
@@ -17,6 +17,12 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   pkg-config \
   mingw-w64
 
+RUN curl -sL https://nodejs.org/dist/v14.4.0/node-v14.4.0-linux-x64.tar.xz | \
+  tar -xJ
```

Comment:
Nit, our indentation here is quite messy. We have 2 spaces, 4 spaces and even tabs.